peret/__init__.py

```diff
@@ -117,7 +117,11 @@ def _has_translation(e: TagNode, lang: str, value: str) -> bool:
     for quote in e.css_select(
         f'entry > sense > cit[type="translation"][xml|lang="{lang}"] > quote'
     ):
-        if value in quote.full_text:
+    return e.css_select(
+        f'entry > sense > cit[type="translation"][xml|lang="{lang}"] > quote'
+    ).filtered_by(
+        lambda quote: quote.full_text == value
+    ).size > 0
             return True
```

Comment:
```suggestion
        return e.css_select(
            f'entry > sense > cit[type="translation"][xml|lang="{lang}"] > quote'
        ).filtered_by(
            lambda quote: quote.full_text == value
        ).size > 0
```

Reply:
this?

Reply:
dang it. go that at home.

Reply:
at least i could break the interface. or vice versa.

Reply:
yes please, that is what i meant!  i don't really get how multiline suggestions work on github.